<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter</class>
 <widget class="QMainWindow" name="filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_wgt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>411</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="archive_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Архив</string>
    </property>
   </widget>
   <widget class="QLabel" name="filter_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фильтр</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_txt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>32</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_txt">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>32</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QLabel" name="age_txt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>32</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Возраст</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_pole">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_pole">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="age_pole">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
